doc/commands.md

```diff
@@ -234,8 +215,8 @@ This table contains:
 This may look like:
 
 ```
-problem               yaml ini tex sol    Ival Oval    sample secret    AC  WA TLE    cpp java py2 py3   comment
-A appealtotheaudience    Y   Y   Y   N       Y    Y         2     30     4   4   2      2    0   0   2
+problem               time yaml tex sol   val: I A O    sample secret bad    AC  WA TLE subs   cpp py java kt   comment
```

Comment:
side note: maybe `bad` can be replaced by `inv: I A O` just like `val`. Although maybe that's more verbose than needed.

Reply:
The `bad` stat is currently the sum of all invalid directories. I think that's fine, I don't use it much anyway :stuck_out_tongue: 

(note that I updated `bt stats` on `master`, thanks @mzuenni for updating the documentation :smile: :heart:)

Reply:
At least we can rename to `invalid` or `inv` now